src/nupic/algorithms/KNNClassifier.py

```diff
@@ -201,6 +207,7 @@ class labels
     self.replaceDuplicates = replaceDuplicates
     self.cellsPerCol = cellsPerCol
     self.maxStoredPatterns = maxStoredPatterns
+    self.minSparsity = minSparsity
```

Comment:
@oxtopus  I was hoping I didn't need to worry about backward compatibility here since this class is not used much outside of research, but I can do that.